src/mmw/apps/modeling/calcs.py

```diff
@@ -36,3 +39,42 @@ def animal_population(geojson):
         'name': 'animals',
         'categories': aeu_return_values
     }
+
+
+def point_source_pollution(geojson):
+    """
+    Given a GeoJSON shape, call MapShed's `point_source_discharge` method
```

Comment:
It doesn't look like we are calling MapShed's [`point_source_discharge`](https://github.com/WikiWatershed/model-my-watershed/blob/c2eacd3383753a51badfed30190dab33e3e4e03b/src/mmw/apps/modeling/mapshed/calcs.py#L351) method here?